scripts/ci/cmake-v2/ci-ubuntu20.04-clang10-mpi.cmake

```diff
@@ -0,0 +1,41 @@
+include(ProcessorCount)
+ProcessorCount(NCPUS)
+math(EXPR N2CPUS "${NCPUS}*2")
+
+set(ENV{CC}  clang-10)
+set(ENV{CXX} clang++-10)
+set(ENV{FC}  gfortran-11)
+
+execute_process(
+  COMMAND "python3-config" "--prefix"
+  OUTPUT_VARIABLE PY_ROOT
+  OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+set(dashboard_cache "
+ADIOS2_USE_BZip2:BOOL=ON
+ADIOS2_USE_Blosc:BOOL=ON
+ADIOS2_USE_DataMan:BOOL=ON
+ADIOS2_USE_Fortran:BOOL=ON
+ADIOS2_USE_HDF5:BOOL=ON
+ADIOS2_USE_MPI:BOOL=ON
+ADIOS2_USE_Python:BOOL=ON
+ADIOS2_USE_SZ:BOOL=ON
+ADIOS2_USE_ZeroMQ:STRING=ON
+ADIOS2_USE_ZFP:BOOL=ON
+
+Python_ROOT_DIR:PATH=${PY_ROOT}
+Python_FIND_STRATEGY:STRING=LOCATION
+Python_FIND_FRAMEWORK:STRING=FIRST
+
+CMAKE_C_FLAGS:STRING=-Wall
+CMAKE_CXX_FLAGS:STRING=-Wall
+CMAKE_Fortran_FLAGS:STRING=-Wall
+
+MPIEXEC_EXTRA_FLAGS:STRING=--allow-run-as-root --oversubscribe
+MPIEXEC_MAX_NUMPROCS:STRING=${N2CPUS}
+")
+
+set(CTEST_TEST_ARGS PARALLEL_LEVEL 1)
+set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
+list(APPEND CTEST_UPDATE_NOTES_FILES "${CMAKE_CURRENT_LIST_FILE}")
+include(${CMAKE_CURRENT_LIST_DIR}/ci-common.cmake)
```

Comment:
We need to merge the files from cmake-v2 to the cmake folder giving priority to the files from cmake-v2.

Reply:
I think I'm missing something, do you mean we need that eventually?  Or as a part of this PR?  What is the goal there, just cleaning up old stuff we don't use anymore?

Reply:
I was referring to doing it in this PR but yeah lets have this merge soon, lets do this in another PR.